tests/tests_test_workflow/test_integ_workflow/integ_test/data/build_manifest_osd.yml

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
Better to add tests data here: https://github.com/opensearch-project/opensearch-build/tree/main/tests/data ?

Reply:
This is in sync with the opensearch side of the data location.
It might be a lot more changes if we want to move.
But those test data are integTest specific so I havent touch them yet. Can move later. Thanks.